### PR TITLE
Add ruby_namespace option to VideoIntelligence

### PR DESCRIPTION
**Pull request by blowmage**

### CODE REVIEW

File: google/cloud/videointelligence/v1/video_intelligence.proto [hunk 1]
```diff
@@ -28,6 +28,7 @@ option java_multiple_files = true;
 option java_outer_classname = "VideoIntelligenceServiceProto";
 option java_package = "com.google.cloud.videointelligence.v1";
 option php_namespace = "Google\\Cloud\\VideoIntelligence\\V1";
+option ruby_namespace = "Google::Cloud::VideoIntelligence::V1";
```
Comment (andreamlin) [inline google/cloud/videointelligence/v1/video_intelligence.proto]: Is this going to change the existing Ruby proto package name?

Comment (blowmage) [reply google/cloud/videointelligence/v1/video_intelligence.proto]: No. The gem name will stay the same. The generated GAPIC client code is already using VideoIntelligence. This change will resolve an outstanding issue about confusing docs.

https://github.com/googleapis/google-cloud-ruby/issues/1762

Comment (andreamlin) [reply google/cloud/videointelligence/v1/video_intelligence.proto]: Great!

**Comment by blowmage** [comment]
Can we get an update on this?

**Comment by chrisdunelm** [comment]
Old PR. Closing.

**Pull request closed without merging**